### DISCUSSION TITLE
Welcome to the iot-cardboard-js community! 👋

### DISCUSSION THREAD

**Discussion by ccrowley96** [microsoft/iot-cardboard-js#118] [Announcements]
## Welcome to `iot-cardboard-js` 
Feel free to ask questions and share ideas for this repo here.

## Issue templates
[File a bug](https://github.com/microsoft/iot-cardboard-js/issues/new?assignees=&labels=bug+%3Abug%3A&template=bug-report.md&title=) 🐛
[Request a new feature](https://github.com/microsoft/iot-cardboard-js/issues/new?assignees=&labels=enhancement+%3Abulb%3A&template=feature_request.md&title=) 💡

**Comment by Sarthak2426** [comment]
thank you sir for such data. i need your help can u pls help me out how to get the react_scene _id

**Reply by Sarthak2426** [reply]
its a humble request.. its shows this only

![image](https://user-images.githubusercontent.com/90612035/201489026-bde0bc8d-d10e-4482-a969-8fe600be2f19.png)

**Reply by Sarthak2426** [reply]
![image](https://user-images.githubusercontent.com/90612035/201489064-5aa38d1e-d58f-4f08-8f8b-b623c1b4cd56.png)

**Reply by hemanth2820** [reply]
Hey Sarthak!!
Did you get the solution for the Scene Id?

**Comment by hemanth2820** [comment]
Hey @ccrowley96, I tried the same model. I am not able to see the scenes when running my model in local machine. It's saying **Cors Header issue**. I tried all possible ways to resolve it. Still facing same issue. Could you suggest any way to resolve this issue.
<img width="958" alt="cors error" src="https://user-images.githubusercontent.com/110812003/230302778-cd2ab2b1-40fa-4c1e-b764-69453ec65fb3.PNG">

**Comment by Sarthak2426** [comment]
i am working on it

**Reply by hemanth2820** [reply]
I would like to know more about it. Can you please provide your Mail ID and can we please connect at your convenient time.

**Comment by Sarthak2426** [comment]
sure sir. Mail id. ***@***.***

**Reply by hemanth2820** [reply]
@Sarthak2426 Hey Sarthak, I can't see your mail id. It's encoded with **. Can you please provide your valid mail id

**Comment by BurmaByte** [comment]
![MicrosoftTeams-image (12)](https://github.com/microsoft/iot-cardboard-js/assets/137844689/c383882e-6bfe-45b4-bf55-ccd61f82a822)
How could I overcome that error? Tired to autheticate from AAD but still showing that error message.
